chart/templates/dashboards/ping-dashboard.yaml

```diff
@@ -22,7 +22,19 @@ data:
       "gnetId": null,
       "graphTooltip": 0,
       "id": 1,
-      "links": [],
+      "links": [
+          {
+            "asDropdown": true,
+            "icon": "cloud",
+            "includeVars": false,
+            "keepTime": false,
+            "tags": [],
+            "targetBlank": true,
+            "title": "dNation - Making Cloud Easy",
+            "type": "link",
+            "url": "https://www.dNation.cloud/"
+          }
+        ],
```

Comment:
Apply the same change in the [docker dashboard](https://github.com/dNationCloud/ping/blob/main/docker/grafana/dashboards-json/Ping.json)